contracts/minter/src/contract_tests.rs

```diff
@@ -1462,6 +1462,97 @@ fn unhappy_path() {
     assert!(res.is_err());
 }
 
+//TODO for debug to test shuffle. remove before prod
+#[test]
+fn shuffle() {
+    const SHUFFLE_FEE: u128 = 100_000_000;
+    // setup accounts
+    let mut router = custom_mock_app();
+    let (creator, buyer) = setup_accounts(&mut router);
+    // setup contracts
+    let num_tokens = 20;
+    let (minter_addr, _config) = setup_minter_contract(&mut router, &creator, num_tokens);
+    // query mintable order for mints
+    let query_mintable_tokens_msg = QueryMsg::MintableTokens {};
+    let res: MintableTokensResponse = router
+        .wrap()
+        .query_wasm_smart(minter_addr.clone(), &query_mintable_tokens_msg)
+        .unwrap();
+    println!("{:?}", res);
```

Comment:
Isn't it nicer to use something like `log::debug!()` instead of `println!`? Just wondering.

Reply:
I have used ```dbg!()``` macro

Reply:
`dbg!()` sounds like a good option too